app/javascript/react/screens/App/Plan/Plan.js

```diff
@@ -127,7 +141,13 @@ class Plan extends React.Component {
             <Link to="/migration">{__('Migration')}</Link>
           </li>
           {!isRejectedPlan &&
-            planName && <Breadcrumb.Item active>{planName}</Breadcrumb.Item>}
+            planName && (
+              <Breadcrumb.Item active>
+                {breadcrumbIcon}
```

Comment:
golly `&nbsp;` makes me cringe... what about doing a template literal here instead?  🤔 (or not and I'll keep cringing)

Reply:
`Yeah,&nbsp;they&nbsp;are&nbsp;ugly&nbsp;aren't&nbsp;they.&nbsp;&nbsp;`

as much as i want to keep making you cringe, i'll use a literal. good point. I figured this was better than special margin CSS, but I forgot i could specify spaces more cleanly, so thanks for that tip @AllenBW

Reply:
`&nbsp;  4 L1F3 &trade;`

Reply:
to be honest i wish spaces looked more like this: `gl.createBuffer(); gl.bufferData([1.0, -1.0,  1.0, 1.0, -1.0,-1.0, -1.0,]` - but hey, who's being picky here? 🤣 

+1 for the template literal if that's easier to see in the JSX... i just thought you guys liked to kick it old-school... 🎶 📻 🎶